Description: Fix version
 This patch must be changed on each new upstream release.
Author: Alexander Golovko <alexandro@ankalagon.ru>
Last-Update: 2012-12-17

diff --git a/VERSION b/VERSION
index dedcc7d..d4a2deb 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-4.0.0
+4.0.0-4
diff --git a/config/initializers/2_app.rb b/config/initializers/2_app.rb
index fc97960..3bd10ce 100644
--- a/config/initializers/2_app.rb
+++ b/config/initializers/2_app.rb
@@ -1,6 +1,6 @@
 module Gitlab
   Version = File.read(Rails.root.join("VERSION"))
-  Revision = `git log --pretty=format:'%h' -n 1`
+  Revision = '6a932d0'
 
   def self.config
     Settings 
